src/netconf_subscribed_notifications.h

```diff
@@ -21,13 +21,26 @@
 #include <sysrepo.h>
 
 #include "common.h"
+#include "receivers.h"
+
+/**
+ * @brief state of a configured subscribed-notifications.
+ */
+enum configured_subscription_state {
+    CONFIGURED_SUBSCRIPTION_STATE_NONE = 0,
+    CONFIGURED_SUBSCRIPTION_STATE_VALID,
+    CONFIGURED_SUBSCRIPTION_STATE_INVALID,
+    CONFIGURED_SUBSCRIPTION_STATE_CONCLUDED
```

Comment:
I would like some abbreviation and maybe even a common prefix. What about `enum sub_cfg_state`? Then the values would be `SUB_CFG_STATE_NONE`, ...